### PR TITLE
Upload capacity tracker data

### PR DESCRIPTION
**Pull request by MarionJHolloway**
# Description
A script to import care home and non-residential capacity tracker csv files to two parquet datasets.

# How to test
Unit tests passing
Run in branch - behaving as expected

# Developer checklist
- [ ] Unit test
- [ ] Linked to Trello ticket
- [ ] Documentation up to date

### CODE REVIEW

File: jobs/ingest_capacity_tracker_data.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    df = utils.read_csv_with_defined_schema(source, schema)
+    df = add_column_with_formatted_dates(
+        df,
+        date_column_name,
+        date_format,
+    )
+    utils.write_to_parquet(df, destination, False)
+
+
+def add_column_with_formatted_dates(df, old_column, date_format):
```
Comment (garypolzin) [inline jobs/ingest_capacity_tracker_data.py:41]: change old_column to be more descriptive, the function above has date_column_name which would work

File: jobs/ingest_capacity_tracker_data.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import pyspark.sql.functions as F
+
+from schemas.capacity_tracker_schema import (
+    CAPACITY_TRACKER_CARE_HOMES,
+    CAPACITY_TRACKER_NON_RESIDENTIAL,
+)
+from utils import utils
+
+
+def main(care_home_source, care_home_destination, non_res_source, non_res_destination):
```
Comment (garypolzin) [inline jobs/ingest_capacity_tracker_data.py:12]: are you able to replicate the destination path from another job, such as ingest_cqc_pir_data
that job automatically constructs the destination file path

**Review by garypolzin — CHANGES_REQUESTED** [review]
nice, neat and easy to follow but I think the auto generation of the destination will be advantageous if we plan to do this frequently. If it's going to be fairly infrequent then you could argue we don't need it

**Comment by MarionJHolloway** [comment]
I've made the requested changes. Everything is running happily in branch.

**Review by garypolzin — APPROVED** [review]

**Pull request merged**